### ISSUE TITLE
From Sketch to Figma

### ISSUE THREAD

**Issue by 2amarkina** [blockframes/blockframes#7342]
### Preparation

This epic aims to move little by little the components from Sketch to Figma, in order to maybe abandon Sketch in the future. 
Some research has been already done on this matter, but we need to be sure about the following: 

- [ ] Check out Figma and Zeplin Integration 
- [ ] Compare prices for Sketch and Figma 

The research that has been done shows the following advantages of moving from Sketch to Figma: 
- Ability to work with any OS (Mac, Windows etc...) ; 
- Better synchronization (Sketch is getting hard to maintain) ; 
- Gives us a better cleaning possibility ; 
- One tool for whole design team. 

**Ressources:** 
- [Figma vs Sketch important links](https://www.figma.com/figma-vs-sketch/)
- [Migrating to Figma](https://www.figma.com/best-practices/migrating-to-figma/)
- [How to process for the migration](https://www.figma.com/best-practices/what-to-expect-when-moving-from-sketch-to-figma/)
- [Details on what can happen ](https://help.figma.com/hc/en-us/articles/360040514273)

### Execution
_If go for it, then_ 
- Take an existing component ;
- Migrate to Figma ; 
- Check the missing styles ; 

### Clean up

Need to take into account the actual files on Zeplin + Sketch and prepare the new ground on Figma at the same time. 

### Estimation

Should be done little by little, during the cleaning time. _The research should be done before._

### Priority

Low

### Anything else?

This task should be executed only when there is no urgent tasks to do. _The research should be done before._

**Comment by mriiad** [comment]
A Figma design could be a very useful thing for frontend developers, especially on how the web page is organized and how components and their children should/must be created.

But Sketch is not a bad thing and it has its advantages too. In my opinion, using both is the best way to have a perfect final product because Sketch is a good way to express customer's needs before moving to a design (Figma), so it could be useful but I suggest not to spend a lot of time on it.

**Comment by stephanie-cascade8** [comment]
Work started [here](https://www.figma.com/files/project/45794552/UI-Library?fuid=1009015793376636421)
